form-val.js

```diff
@@ -10,3 +10,30 @@ function validate(e) {
     err.innerHTML = 'Email cannot be uppercase';
   }
 }
+
```

Comment:
- [ ] [OPTIONAL] Your form is not styled right kindly consider fixing this so your design looks professional 

![Screenshot_20220625_183033](https://user-images.githubusercontent.com/25227022/175784360-7b0c8a30-c8f1-4542-ba56-409b4e56f9b5.png)